ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java

```diff
@@ -854,15 +796,12 @@ static UnsignedLong base_reward(
    * @param previous_total_balance
    * @return
    */
-  static UnsignedLong inactivity_penality(
-      BeaconState state,
-      int index,
-      UnsignedLong epochs_since_finality,
-      UnsignedLong previous_total_balance) {
+  static long inactivity_penality(
+      BeaconState state, int index, long epochs_since_finality, long previous_total_balance) {
     return base_reward(state, index, previous_total_balance)
```

Comment:
The order of operations on this statement don't match between the UL and long versions. Double check the spec and make sure the order of operations is correct.

Reply:
It is correct